docs/blog/clever-release.md

```diff
@@ -0,0 +1,20 @@
+# MakeCode Clever Integration Released!
+Just in time for the fall semester, MakeCode has integrated with [Clever](https://www.clever.com/)!
```

Comment:
I might say something like "Just in time for back to school, MakeCode is adding more ways for students and teachers to sign in!  [Clever](https://www.clever.com) is an education platform that provides identity and security services, making it easy to access classroom resources all in one place."